data/meetups.yml

```diff
@@ -102,19 +101,23 @@
     image:    22barcelona.png
   - location: Kyiv, Ukraine
     url:      http://ember-club.kiev.ua
-  - location: Belo Horizonte, Brazil
-    url:      http://www.meetup.com/Ember-BH/
   - location: Zürich, Switzerland
     url:      http://www.meetup.com/Ember-js-Zurich/
-  - location: Vancouver, BC
-    url:      http://www.meetup.com/Vancouver-Ember-js/
   - location: Amsterdam, the Netherlands
     url:      http://www.meetup.com/Amsterdam-Ember-js/
+"Asia":
+  - location: Tel Aviv, Israel
+    url:      http://www.meetup.com/Ember-IL/
+    image:    20telaviv.png
   - location: Bangalore, India
     url:      http://www.meetup.com/EmberJS-Bangalore/
   - location: Tokyo, Japan
     url:      http://emberjs.doorkeeper.jp/
+"Australia":
```

Comment:
[Oceania](http://en.wikipedia.org/wiki/Oceania) seems the correct term for the region.